### PR TITLE
Release notes for v0.1.24

### PR DESCRIPTION
**Pull request by andyjih**
Screen Shot 2020-11-17 at 3 34 37 PM

### CODE REVIEW

File: pages/whats-new/2020-11-17-batch-imports-and-mailchimp-source.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Batch Imports and Mailchimp Source"
+date: "2020-11-17"
+tags: ["new", "improvement", "fix"]
+description: "Batch Imports and Mailchimp as a Source"
+github: "https://github.com/grouparoo/grouparoo/releases/tag/v0.1.24"
```
Comment (evantahler) [inline pages/whats-new/2020-11-17-batch-imports-and-mailchimp-source.mdx]: Got the URL right! https://github.com/grouparoo/grouparoo/releases/tag/v0.1.24

**Review by evantahler — APPROVED** [review]

File: pages/whats-new/2020-11-17-batch-imports-and-mailchimp-source.mdx (hunk 1 above)
Comment (andyjih) [reply pages/whats-new/2020-11-17-batch-imports-and-mailchimp-source.mdx]: Predicted! :D

**Comment by andyjih** [comment]
@bleonard ok, split them up!

File: components/home/customerTestimonials.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+              </Link>
+              <br />
+              <p>
+                <em>
+                  "I was able to set up a process that merges and maintains user
+                  profiles with data from multiple sources in just a few minutes
+                  and I wish we had it years ago.
+                </em>
+              </p>{" "}
+              <p>-- Joaquín Brenner, Co- founder and COO @ Chipax</p>
```
Comment (bleonard) [inline components/home/customerTestimonials.tsx]: extra space in there? `Co- founder`

File: pages/whats-new/2020-11-17-batch-imports-and-dashboard.mdx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Batch Imports and Updated Dashboard"
+date: "2020-11-17"
+tags: ["new"]
+description: "Batch Imports and updated dashboard"
+github: "https://github.com/grouparoo/grouparoo/pull/972"
```
Comment (bleonard) [inline pages/whats-new/2020-11-17-batch-imports-and-dashboard.mdx:7]: `image: "releases/2020-11-17-dashboard.png"`

File: pages/whats-new/2020-11-17-mailchimp-as-a-source.mdx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Mailchimp as a Source"
+date: "2020-11-17"
+tags: ["new"]
+description: "Mailchimp as a Source"
+github: "https://github.com/grouparoo/grouparoo/pull/969"
+---
+
+You can now add Mailchimp as a Source. Some of our users were asking for a way to identify users in Mailchimp using a property besides email, so now you can import users from Mailchimp into Grouparoo and identify them using any unique identifier that exists in Mailchimp.
```
Comment (bleonard) [inline pages/whats-new/2020-11-17-mailchimp-as-a-source.mdx]: Might be useful to mention you can now pull in click and open rates.

File: pages/whats-new/2020-11-17-sqlite.mdx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Grouparoo can run off of SQLite"
+date: "2020-11-17"
+tags: ["new"]
+description: "Grouparoo can run off of SQLite"
+github: "https://github.com/grouparoo/grouparoo/pull/965"
+---
+
+Now you can choose between SQLite and Postgres for the database system underlying Grouparoo. You can make these modifications in your environment variables.
```
Comment (bleonard) [inline pages/whats-new/2020-11-17-sqlite.mdx]: Maybe an example on postgres and how to set it to SQLite?

**Review by bleonard — APPROVED** [review]

**Pull request merged**